.github/ISSUE_TEMPLATE/new-environment.yml

```diff
@@ -0,0 +1,209 @@
+
+        The is-production tag will be inferred from the environment and is not needed here
+  - type: input
+    id: tag1
+    attributes:
+      label: application 
+      value: 
+    validations:
+      required: true
+  - type: input
+    id: tag2
+    attributes:
+      label: business-unit 
+      value: 
+    validations:
+      required: true
+  - type: input
+    id: tag3
+    attributes:
+      label: infrastructure-support 
+      value: 
+    validations:
+      required: true
+  - type: input
+    id: tag4
+    attributes:
+      label: owner 
+      value: 
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: "Valid business-unit values | HQ,HMPPS,OPG,LAA,HMCTS,CICA,Platforms,CJSE | The infrastructure-support tag should be an email address which will receive AWS Health Operations emails."
+  - type: markdown
+    attributes:
+      value: "## Networking options"
+  - type: checkboxes
+    id: subnet-sets
+    attributes:
+      label: Subnet sets
+      description: Please choose one of the below, most applications will use the general subnet set for their business unit. This means that they will benefit from out of the box connectivity to other applications, most applications will use the general subnet.  If an application has highly sensitive data it may need to go into a subnet with limited connectivity.
+      options:
+        - label: General
+        - label: Isolated
+    validations:    
+      required: true
+  - type: dropdown
+    id: app-connect
+    attributes:
+      label: How do users connect to the application
+      description: 
+      multiple: false
+      options:
+        - Over the public internet
+        - With a purple cabled device (please give details)
+        - With a MoJ Official device
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: "### Connectivity to other applications or external parties"
+  - type: markdown
+    attributes:
+      value: Please detail here and connectivity that your application needs, eg to other applications or external parties
+  - type: textarea
+    id: add-feat
+    attributes:
+      label: Additional features
+      description: For more information see here https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/creating-networking.html#certificate-services If you are not sure you can leave these blank and they can be added at a later date
+      value:
+
+        Please check any additional features required 
+
+        - [ ] Additional VPC Endpoints
+
+        - [ ] Extended DNS Zones
+
+        - [ ] Other - please specify
+
+    validations:
+      required: true
+  - type: textarea
+    id: other-info
+    attributes:
+      label: Other information
+      description: Any other information you feel is relevant, please remember this is a public repository
+      value:
+    validations:
+      required: 
+  - type: textarea
```

Comment:
Would this work better as a checkbox? or should it stay as a text box so additional requirements could be added? 🤔